src/controllers/portfolio/portfolio.ts

```diff
@@ -109,6 +75,11 @@ export class PortfolioController extends EventEmitter {
 
   #additionalHints: GetOptions['additionalHints'] = []
```

Comment:
We can delete these as they aren't used anymore.

Also, we can delete `resetAdditionalHints` and the places we use it